app/experimenter/legacy-ui/rapid/contexts/experiment/actions.ts

```diff
@@ -1,21 +1,36 @@
 import {
   ExperimentReducerActionType,
   ExperimentData,
+  ExperimentAnalysis,
 } from "experimenter-types/experiment";
 import { ExperimentReducerAction } from "experimenter-types/experiment";
 
 export const fetchExperiment = (experimentSlug: string) => async (
   experimentData: ExperimentData,
   dispatch: React.Dispatch<ExperimentReducerAction>,
 ): Promise<void> => {
-  const response = await fetch(`/api/v3/experiments/${experimentSlug}/`, {
+  const data_response = await fetch(`/api/v3/experiments/${experimentSlug}/`, {
```

Comment:
I might use the word `experiment` here instead of `data`?

Reply:
Oooh good catch